### PR TITLE
Resolve linter issues (ansible-lint, yamllint, flake8)

### LINKED ISSUE #308: Resolve all linter issues (ansible-lint, yamllint, flake8)
## Issue Description
We have discovered that our codebase has a number of issues detected by various linters: ansible-lint, yamllint, and flake8. In order to maintain high code quality and consistency across the project, we need to address these issues.

## Task

- [ ] Review and fix all ansible-lint issues
- [ ] Review and fix all yamllint issues
- [ ] Review and fix all flake8 issues

## Additional Information

1. ansible-lint: This linter checks the Ansible playbooks and roles for best practices and potential errors. For more information, please check the [ansible-lint documentation](https://ansible-lint.readthedocs.io/en/latest/).

2. yamllint: This linter checks YAML files for syntax errors, indentation, and other stylistic issues. For more information, please check the [yamllint documentation](https://yamllint.readthedocs.io/en/stable/).

3. flake8: This linter checks Python code for compliance with PEP8, syntax errors, and other code quality issues. For more information, please check the [flake8 documentation](https://flake8.pycqa.org/en/latest/).

Please make sure to configure the linters with the appropriate configuration files and rules for our project. Submit a pull request with the necessary changes and fixes, and make sure the continuous integration pipeline passes without any issues.

### PR DESCRIPTION
**Pull request by ThomasSanson**
Close #308

### CODE REVIEW

**Comment by vitabaks** [comment]
@ThomasSanson consider offering a PR for yedit.py on the project [repository](https://github.com/kwoodson/ansible-role-yedit)

**Comment by ThomasSanson** [comment]
Hi @vitabaks, thank you for your comment. 

I'm not quite sure if I understood your request correctly. 

Are you suggesting that I should I cancel the changes I made in the current Pull Request ? 

Please let me know so I can proceed accordingly. Thanks !

**Comment by vitabaks** [comment]
not necessarily cancel these changes.

It would just be nice to suggest improvements to the original repository of this module as well.

**Comment by vitabaks** [comment]
in fact, this is not a role, but a library for managing the yaml configuration, which you can observe here https://github.com/vitabaks/postgresql_cluster/tree/master/roles/patroni/library

**Comment by ThomasSanson** [comment]
@vitabaks, I'm back from vacation ^^

I have just made the corrections for the merge conflicts, and everything is good to go for me to merge this pull request.

**Comment by vitabaks** [comment]
@ThomasSanson Welcome back! I'm glad you're here)

please don't forget to remove the exceptions from the [.ansible-lint](https://github.com/vitabaks/postgresql_cluster/blob/master/.ansible-lint) based on what has been fixed. 

To have a clear understanding of what else needs attention.

**Comment by ThomasSanson** [comment]
Hi @vitabaks,

Thank you for your comment.

I understand your concern regarding the skip_list rules in .ansible-lint. For now, I have just made the linters pass for this pull request.

My plan is to create a separate issue for each rule in skip_list after this pull request. This will allow us to address each issue systematically and have a clear understanding of what else needs attention.

I hope this addresses your concerns. Thank you again for your feedback.

File: config_pgcluster.yml [hunk 1]
```diff
@@ -133,18 +133,20 @@
...
     - role: copy
 
-- import_playbook: balancers.yml
+- name: config_pgcluster.yml | Configure HAProxy Load Balancing
+  import_playbook: balancers.yml
   when: with_haproxy_load_balancing|bool
   tags: config_balancers,load_balancing, haproxy
 
-- hosts: postgres_cluster
+- name: config_pgcluster.yml | Deploy Postgres Cluster
```
Comment (vitabaks) [inline config_pgcluster.yml:141]:
```suggestion
- name: config_pgcluster.yml | Configure Postgres Cluster
```

File: deploy_pgcluster.yml [hunk 2]
```diff
@@ -1,13 +1,12 @@
 ---
-
-- name: Deploy PostgreSQL HA Cluster (based on "Patroni" and "{{ dcs_type }}")
+- name: deploy_pgcluster.yml | Deploy PostgreSQL HA Cluster (based on "Patroni" and "{{ dcs_type }}")
```
Comment (vitabaks) [inline deploy_pgcluster.yml:2]:
```suggestion
- name: Deploy PostgreSQL HA Cluster (based on "Patroni" and "{{ dcs_type }}")
```
why is the playbook name added here?

Comment (ThomasSanson) [reply deploy_pgcluster.yml]: I have incorporated the playbook names into the task names to enhance readability in the logs. However, it's important to note that this change is a proactive step towards our planned migration to collection mode

**Pull request merged**